src/plots/barPlot.ts

```diff
@@ -500,7 +500,7 @@ export class Bar<X, Y> extends XYPlot<X, Y> {
 
       let barWidth = attrToProjector["width"](d, i, dataset);
       let barHeight = attrToProjector["height"](d, i, dataset);
-      let text = this._labelFormatter(valueAccessor(d, i, dataset));
+      let text = this._labelFormatter(valueAccessor(d, i, dataset), d, [dataset]);
```

Comment:
That would be ideal, yes. An array of datasets is needed because of one use case: formatting the label of stacked bar chart totals. In that case the value is not tied to a single dataset but rather all the datasets that have a value at that axisValue. For example, if the chart is displaying various metrics grouped by state name we need to know all the datasets that were used to compute the total for each state name.

Because of this the API is constrained to be `Dataset[]`. Unfortunately most formatting use cases are for points on a single dataset we see `[dataset]` in various places.